app/src/main/java/com/cortado/mafkir/ui/MainActivity.kt

```diff
@@ -22,4 +28,14 @@ class MainActivity : DaggerAppCompatActivity() {
 
         startService(Intent(this, InteractionsService::class.java))
     }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
```

Comment:
why do you need this?

Reply:
for the global "back" functionality

Reply:
i commented the code and every thing works the same... what wont work without it??

Reply:
dn